iis-common/src/main/java/eu/dnetlib/iis/common/java/io/SequenceFileTextValueReader.java

```diff
@@ -0,0 +1,168 @@
+package eu.dnetlib.iis.common.java.io;
+
+import java.io.IOException;
+import java.util.regex.Pattern;
+
+import org.apache.hadoop.fs.FileSystem;
+import org.apache.hadoop.fs.LocatedFileStatus;
+import org.apache.hadoop.fs.RemoteIterator;
+import org.apache.hadoop.io.SequenceFile;
+import org.apache.hadoop.io.SequenceFile.Reader;
+import org.apache.hadoop.io.Text;
+import org.apache.hadoop.io.Writable;
+import org.apache.hadoop.util.ReflectionUtils;
+import org.apache.log4j.Logger;
+
+/**
+ * Iterator that extracts sequence file's consecutive {@link Text} values. 
+ * 
+ * @author mhorst
+ */
+public class SequenceFileTextValueReader implements CloseableIterator<Text> {
+
+        private final Logger log = Logger.getLogger(this.getClass());
+
+        private SequenceFile.Reader sequenceReader;
+
+        private final RemoteIterator<LocatedFileStatus> fileIt;
+
+        private final FileSystem fs;
+
+        /** Ignore file starting with underscore. Such files are also ignored by
+         * default by map-reduce jobs.
+         */
+        private final static String whitelistRegexp = "^(?!_).*";
```

Comment:
Precompile with `Pattern.compile`?
Also "^[^_].*" could be more understandable at first sight (and only differ in not matching an empty string).